templates/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -0,0 +1,136 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod`.telemetry_derived.experiment_enrollment_aggregates_live AS (
+  WITH live AS (
+    SELECT
+      event_object AS `type`,
+      event_string_value AS experiment,
+      `moz-fx-data-shared-prod`.udf.get_key(event_map_values, 'branch') AS branch,
+      TIMESTAMP_ADD(
+        TIMESTAMP_TRUNC(`timestamp`, HOUR),
+        INTERVAL (
+          CAST(
+            EXTRACT(
+              MINUTE
+              FROM
+                `timestamp`
+            ) / 5 AS int64
+          ) * 5
+        ) MINUTE
+      ) AS window_start,
+      TIMESTAMP_ADD(
+        TIMESTAMP_TRUNC(`timestamp`, HOUR),
+        INTERVAL (
+          CAST(
+            EXTRACT(
+              MINUTE
+              FROM
+                `timestamp`
+            ) / 5 + 1 AS int64
+          ) * 5
+        ) MINUTE
+      ) AS window_end,
+      COUNTIF(event_method = 'enroll') AS enroll_count,
+      COUNTIF(event_method = 'unenroll') AS unenroll_count,
+      COUNTIF(event_method = 'graduate') AS graduate_count,
+      COUNTIF(event_method = 'update') AS update_count,
+      COUNTIF(event_method = 'enrollFailed') AS enroll_failed_count,
+      COUNTIF(event_method = 'unenrollFailed') AS unenroll_failed_count,
+      COUNTIF(event_method = 'updateFailed') AS update_failed_count
+    FROM
+      `moz-fx-data-shared-prod`.telemetry_derived.events_live
+    WHERE
+      event_category = 'normandy'
+      AND date(timestamp)
+  > @submission_date
+  GROUP BY
+    1,
+    2,
+    3,
+    4,
+    5
```

Comment:
Since BigQuery allows that, I would use column aliases here for clarity and to avoid errors during refactors later.